tests/python/RadonBearTest.py

```diff
@@ -25,15 +25,11 @@ class class1():
     RadonBear,
     valid_files=(test_file1, test_file2, test_file3),
     invalid_files=(),
-    settings={'radon_ranks_info': '',
-              'radon_ranks_normal': '',
-              'radon_ranks_major': ''})
+    settings={'cc_threshold': 52})
 
 
 RadonBearReportsTest = verify_local_bear(
     RadonBear,
     valid_files=(),
     invalid_files=(test_file1, test_file2),
-    settings={'radon_ranks_info': '',
-              'radon_ranks_normal': 'A',
-              'radon_ranks_major': ''})
+    settings={'cc_threshold': 1})
```

Comment:
Add one more test or change one test to check without passing anything in the settings as you have already passed default `cc_threshold: int=10`.

Reply:
`RadonBearDefaultsTest ` does the same. Check the complete file 😄